frontend/test/cypress/integration/product-list/newsletter_subscribe.spec.ts

```diff
@@ -25,10 +25,8 @@ describe('subscribe to newsletter', () => {
       });
       user.findByLabelText(/enter your email/i).type('test@example.com');
       user.findByRole('button', { name: /subscribe|join/i }).click();
-      user
-         .findByRole('button', { name: /subscribed/i })
-         .should('be.visible')
-         .should('be.disabled');
+      user.findByText('Subscribed!').should('be.visible');
+      user.findByRole('button', { name: /subscribed/i }).should('not.exist');
```

Comment:
This might make sense as a regression test, but now that there's no button that says subscribed, we should also make sure the existing subscribe button is invisible.

Reply:
Resolved 6f8980aa56a05991b0b2f4b2ccdfa006b3e479a1